client/components/ConfirmAuth/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Route from 'react-router-dom';
```

Comment:
This needs to be `import { Route } from 'react-router-dom';`, otherwise it produces an error. Changing it to `import { Route }` fixes the error.